guides/middleware/ormdatabases.html

```diff
@@ -0,0 +1,270 @@
+
+<h2>Define Your Model</h2>
+
+<p>
+    We are going to create a struct to represent the data we will store in the database.
+    We then make it conform to Model (Which itself conforms to Codable) so it can be used with the ORM.
+</p>
+
+<p>At the end of your Application.swift file, create struct representing a Grade:</p>
+
+<pre><code class="language-swift">struct Grade: Model {
+  var course: String
+  var grade: Int
+}
+</code></pre>
+
+<h2>Set up the Database connection</h2>
+
+<p>
+    In the postInit function of your Application.swift file, create your database connection pool:
+</p>
+<pre><code class="language-swift">let pool = PostgreSQLConnection.createPool(host: "localhost", port: 5432, options: [.databaseName("school")], poolOptions: ConnectionPoolOptions(initialCapacity: 1, maxCapacity: 5, timeout: 10000))
+Database.default = Database(pool)
+</code></pre>
+<p>
+    Beneath your connection pool, create the database table that represents this struct:
+</p>
+<pre><code class="language-swift">do {
+  try Grade.createTableSync()
+} catch let error {
+  // Error
+}
+</code></pre>
+<p>
+    To check that a database table called Grades has been created,
+    run your Kitura server and then use psql from the command-line as follows:
+</p>
+<pre><code>psql school
+SELECT * FROM "Grades";
+</code></pre>
+
+<p>
+    This should print the column names of the Grades table with no data (i.e. no rows).
+</p>
+
+<h2>Persisting our Model in the database</h2>
+
+<p>
+    We are going to create a set of Codable routes that
+    store, retrieve and delete grades from the database.
+</p>
+<p>
+    At the bottom of your postInit function, define your routes:
+</p>
+<pre><code class="language-swift">router.get("/grades", handler: loadHandler)
+router.get("/grades", handler: getOneHandler)
+router.post("/grades", handler: postHandler)
+router.delete("/grades", handler: deleteAllHandler)
+router.delete("/grades", handler: deleteOneHandler)
+</code></pre>
+<p>
+    After your postInit function, define your handlers:
+</p>
+<pre><code class="language-swift">func loadHandler(completion: @escaping ([Grade]?, RequestError?) -> Void ) {
+    Grade.findAll(completion)
+}
+func getOneHandler(id: Int, completion: @escaping (Grade?, RequestError?) -> Void ) {
+    Grade.find(id: id, completion)
+}
+func postHandler(grade: Grade, completion: @escaping (Grade?, RequestError?) -> Void ) {
+    grade.save(completion)
+}
+func deleteAllHandler(completion: @escaping (RequestError?) -> Void ) {
+    Grade.deleteAll(completion)
+}
+func deleteOneHandler(id: Int, completion: @escaping (RequestError?) -> Void ) {
+    Grade.delete(id: id, completion)
+}
+</code></pre>
+
+<h2>Accessing the response on the server</h2>
+
+<p>
+    Because the ORM API matches the Codable routes API,
+    you can pass the completion straight to the Codable route response. <br>
+    You can access your ORM response for your getOneHandler as follows:
+</p>
+
+<pre><code class="language-swift">
```

Comment:
The example code here does not match the route definitions above where getOneHandler calls out to Grade.find(id: id, completion)